active_model_serializers.gemspec

```diff
@@ -42,7 +42,8 @@ Gem::Specification.new do |spec|
   # 'minitest'
   # 'thread_safe'
 
-  spec.add_runtime_dependency 'jsonapi', '0.1.1.beta6'
+  # TODO: Latest jsonapi-renderer is 0.1.2
+  spec.add_runtime_dependency 'jsonapi-renderer', '0.1.1.beta1'
```

Comment:
i didn't want to change the version in this PR. i'd be happy to do another PR that changes the version constraint.